website/docs/developer/development-guide/code/versioning-releases.md

```diff
@@ -0,0 +1,118 @@
+---
+id: versioning-releases
+title:  ORCA Versioning and Releases
+description: Provides information to developers on semantic versioning and the release process.
+---
+
+Much of this documentation is also found at [Cumulus](https://github.com/nasa/cumulus/blob/master/docs/development/release.md).
+
+## Versioning
+
+The ORCA team uses semantic versioning. More information about semantic
+versioning can be found [here](https://semver.org/).
+
+## Release Process
+
+1. **Create a release branch.**
+
+From develop, create a new release branch from develop following the
+`release-MAJOR.MINOR.x`. For example, `release-1.14.1`. Push this branch 
+to github if you created it locally.
+
+2. **Update CHANGELOG.md.**
+
+Update the [CHANGELOG.md](https://github.com/nasa/cumulus-orca/blob/master/CHANGELOG.md). 
+Put a header under the 'Unreleased' section with the new version number and 
+the date.
+
+3. **Create a git tag for the release.**
+
+Make sure you're on the latest commit of the release branch.
+
+Create and push a new git tag:
+
+```
+    git tag -a vx.y.z -m "Release x.y.z"
+    git push origin vx.y.z
+```
+
+4. **Running the deployment**
+
+Publishing of new releases is handled by a Bamboo release plan and is manually
+triggered.
+
+If you created a new release plan in step one, you will need to create a new
+bamboo deployment plan.
+
+**Creating a Bamboo deployment plan**

```

Comment:
This should be an H2 heading I think, maybe H3 or H4 depending on the answer to changing the numbers to headers. So `## Creating a Bamboo deployment plan` instead of  `**Creating a Bamboo deployment plan**`

Reply:
Changed to an H4 heading.